packages/cli/src/commands/local/run.ts

```diff
@@ -31,7 +32,13 @@ export default class Run extends Command {
       this.error(errorMessage, {exit: -1})
     }
 
-    if (flags.env) execArgv.push('--env', flags.env)
+    let envFile = flags.env || '.env'
+    if (fs.existsSync(envFile) && !fs.statSync(envFile).isFile()) {
```

Comment:
This should use `||` instead of `&&` with a boolean not
```
if (!fs.existsSync(envFile) || !fs.statSync(envFile).isFile()) {
```

Reply:
woops, thanks!

Reply:
oh wait, no, this is intentional. so, the way that `node-foreman` works is that you can supply a non-existent file and it will just ignore it. i didn't want to change this behavior. i simply wanted to ensure that if a customer _did_ supply an `env` location, or if the default, .env, was a directory, then ignore it.

Reply:
Yes but in order for `fs.existsSync(envFile)` to pass, you must pass an *existent* path. If the path does not exist, `fs.existsSync(envFile)` will fail and this warning will never be displayed to the user. Is this intentional?

Reply:
e.g If I do `./bin/run local --env "does/not.exist.env` I will not see this warning.

Reply:
yes. that is intentional. the current behavior is such that if the file doesn't exist, node-foreman will ignore it and work fine. i don't see a reason to change that. i specifically only want to change the behavior (and produce the warning message) when the file exists and it is a directory. that's the only case where the cli is currently crashing.

Reply:
Ah! That makes sense. At first glance, it seemed this check was meant to warn the user that the env path they passed in wasn't valid or that the file did not exist. Thank you for the clarification!